lib/tessel/deploy.js

```diff
@@ -99,9 +83,15 @@ const CLI_ENTRYPOINT = 'cli.entrypoint';
   Note that the values are in BYTES!
 */
 
+/**
+ * Retrieve memory information from a Tessel 2.
+ * Language: *
+ *
+ * @return {Promise}
+ */
 Tessel.prototype.memoryInfo = function() {
   return new Promise((resolve, reject) => {
-    return actions.execRemoteCommand(this, 'getMemoryInfo')
```

Comment:
`actions.execRemoteCommand` was just a wrapper for `this.simpleExec`, so all occurrences have been switched over and `actions.execRemoteCommand` removed.